backend/copr_backend/helpers.py

```diff
@@ -240,6 +240,8 @@ def _get_limits_conf(parser):
         parser, "backend", "builds_max_workers_sandbox", 10, mode="int")
     limits['owner'] = _get_conf(
         parser, "backend", "builds_max_workers_owner", 20, mode="int")
+    limits['userssh'] = _get_conf(
+        parser, "backend", "builds_max_userssh", 2, mode="int")
```

Comment:
Might be tricky; we should warn the user that, when the ssh key is specified, only 2 parallel builders will be given to him and that the rest of builds stays in "pending" state.